projects/aca-content/assets/app.extensions.json

```diff
@@ -320,7 +321,7 @@
           {
             "id": "app.create.folderFromTemplate",
             "order": 400,
-            "icon": "create_new_folder",
+            "icon": "adf:create_new_folder",
```

Comment:
why adf prefix here? is that a custom icon?

Reply:
this has been changed to use pre-existing icons